arches_lingo/src/arches_lingo/pages/SchemeReport.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+import Splitter from "primevue/splitter";
+import SplitterPanel from "primevue/splitterpanel";
+import SchemeLicense from "@/arches_lingo/components/scheme/report/SchemeLicense.vue";
+import SchemeNote from "@/arches_lingo/components/scheme/report/SchemeNote.vue";
+import SchemeUri from "@/arches_lingo/components/scheme/report/SchemeUri.vue";
+import SchemeStandard from "@/arches_lingo/components/scheme/report/SchemeStandard.vue";
+import SchemeAuthority from "@/arches_lingo/components/scheme/report/SchemeAuthority.vue";
+</script>
+
+<template>
+    <div>
+        <Splitter>
+            <SplitterPanel>
+                <SchemeNote />
```

Comment:
nit: these don't need to be hardcoded, they can be used in a v-for loop like `SideNav` items